study.md

```diff
@@ -44,21 +44,19 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
-```
+View has to do with what you are seeing, and the ability to have different bits of content presented.  View state relates to storing the page value, or the html, at the time of post back.  I'm not sure what post back means so it is difficult to me to understand what view state is. ```
 
```

Comment:
Awesome! I feel like you have the right idea about view state. You're on the right track about it storing page value. View state is referring to the data from the model that is being displayed to the user in the current view. So if we made an AJAX request and loaded data to our app, that data being displayed is the view state. The front end router is what manages view-state by responding to user inputs, and directing them to the appropriate view!
Take a look at this repo https://github.com/ga-wdi-boston/spa-study/blob/master/study.md !